sdk/communication/azure-communication-administration/azure/communication/administration/_polling.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=W0231
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import base64
+import time
+from typing import Union
+from functools import partial
+import pickle
+
+from azure.core.polling import (
+    PollingMethod
+)
+from ._phonenumber._generated.models import (
+    PhoneNumberSearch,
+    PhoneNumberRelease
+)
+
+class PhoneNumberBasePolling(PollingMethod):
+    """ABC class for reserve/purchase/release phone number related polling.
+    """
+    def __init__(self, is_terminated, interval=5):
```

Comment:
`polling_interval` seems a better name for this param

Reply:
Fixed. Thanks